### PR TITLE
Setup the workflow needed for getting API docs

### PR DESCRIPTION
**Pull request by chaithyagr**
This PR will make sure that we will have runnable API docs which can be triggered.

This must be targeted to `master` so that its effect can take place in every other PR (if I am right).

The example of the output is present in actions in my fork: https://github.com/chaithyagr/pysap-mri/actions/runs/2555701992

### CODE REVIEW

File: .github/workflows/test-docs.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+name: Build API Doc
+
+on: [workflow_dispatch]
+
+jobs:
+  api:
+    name: Deploy API Documentation
+    runs-on: ubuntu-latest
+    env:
+      root: https://github.com/sfarrens/
```
Comment (paquiteau) [inline .github/workflows/test-docs.yml:10]: This should rather be targeting to CEA-COSMIC I guess

File: .github/workflows/test-docs.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      tag: /archive/refs/tags/
+      vpysap: 999
+      folder: pysap-999
```
Comment (paquiteau) [inline .github/workflows/test-docs.yml:13]: why 999? Is it a dummy version number ?

**Review by paquiteau — COMMENTED** [review]
Thanks  for the CI and CD voodoo incantations. I might not be the best to decypher it, but found some minor changes to do.

File: .github/workflows/test-docs.yml (hunk 1 above)
Comment (chaithyagr) [reply .github/workflows/test-docs.yml:10]: Ahhh now maybe yes, initially, it was not yet up

File: .github/workflows/test-docs.yml (hunk 2 above)
Comment (chaithyagr) [reply .github/workflows/test-docs.yml:13]: for now yes.

**Pull request merged**